FND/DatesAndTimes/FinancialDates.rdf

```diff
@@ -692,7 +724,7 @@ The recurrence start date can also be relative to the starting Date of the overa
 		<rdfs:subPropertyOf rdf:resource="&lcc-lr;has"/>
 		<rdfs:label>has duration</rdfs:label>
 		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Duration"/>
-		<skos:definition>the duration of a date period or similar concept, such as the tenor of a contract</skos:definition>
+		<skos:definition>indictes the duration of a date period or similar concept, such as the tenor of a contract</skos:definition>
```

Comment:
typo in first word - though better still delete it

Reply:
replaced with specifies - all properties are arcs in a knowledge graph and their names must start with a verb per our policy.  The only exception is for annotations.